packages/app/ui/beat-list-entry/show-more/description.tsx

```diff
@@ -0,0 +1,22 @@
+import { Image, Text, View } from "app/design-system";
+import { imageUrlMedium } from "app/utils/entry";
+
+export function Description(props: {
```

Comment:
I'd name it differently. `Description` is not well descriptive. Maybe `BeatThumbnail` or `BeatInfo`?